Abalasky tests for events handler

abalasky opened the pull request:
For #1 

"""
First pull request ever so not sure what convention is or what is generally required. Open to feedback. Will detail code changes here...

Also disregard branch name. This is for image forwarding.
"""

Wasn't able to test but I add ImageEvent to domain/domain.go that contains ImageBytes[].

These are obtained from whatsapp.ImageMessage.download() which is called in newly added HandleImageMessage() in events_provider.go

This is then passed to the telegram api and uploaded in similiar manner to QR code.

Hope this works!

dstdfx reviewed (changes requested):
Thanks for the PR, good job working on this. 
I noticed that there's no support for `ImageMessageEvent` in manager, let's add it too: 
https://github.com/dstdfx/twbridge/blob/main/internal/manager/clients_manager.go#L55

abalasky commented:
I addressed all the requested changes and pushed. However when I added the new case in clients_manager.go for ImageMessageEvent I now get a linting error regarding cyclomatic complexity:

internal/manager/clients_manager.go:42:1: cyclomatic complexity 31 of func `(*Manager).Run` is high (> 30) (gocyclo)
func (mgr *Manager) Run(ctx context.Context) {

dstdfx commented:
> I addressed all the requested changes and pushed. However when I added the new case in clients_manager.go for ImageMessageEvent I now get a linting error regarding cyclomatic complexity:
> 
> internal/manager/clients_manager.go:42:1: cyclomatic complexity 31 of func `(*Manager).Run` is high (> 30) (gocyclo) func (mgr *Manager) Run(ctx context.Context) {

We can divide this `Run` method into smaller ones but I'd keep it in a separate PR for the same of simplicity.

For now, we can just fix the limit by adding this to the `.golangcilint.yml`:
```
linters-settings:
  gocyclo:
    min-complexity: 42
```

abalasky commented:
> 

Sorry just returning to this but. I don't see a .golangcilint.yml . Only .golangci.yml. Should i create the dot file or add to .golanci ?

dstdfx commented:
> .golangcilint.yml

You're right, it's called `.golangci.yml`, the piece of configuration that I sent should be located in this file: 
https://github.com/dstdfx/twbridge/blob/main/.golangci.yml
